rmf_site_editor/src/interaction/mod.rs

```diff
@@ -165,14 +168,16 @@ impl Plugin for InteractionPlugin {
                 CategoryVisibilityPlugin::<LiftCabinDoorMarker>::visible(true),
                 CategoryVisibilityPlugin::<LocationTags>::visible(true),
                 CategoryVisibilityPlugin::<FiducialMarker>::visible(true),
-                CategoryVisibilityPlugin::<ConstraintMarker>::visible(true),
                 CategoryVisibilityPlugin::<VisualMeshMarker>::visible(true),
                 CategoryVisibilityPlugin::<CollisionMeshMarker>::visible(false),
                 CategoryVisibilityPlugin::<MeasurementMarker>::visible(true),
                 CategoryVisibilityPlugin::<WallMarker>::visible(true),
-                CategoryVisibilityPlugin::<WorkcellVisualizationMarker>::visible(true),
             ))
-            .add_plugins((CameraControlsPlugin, ModelPreviewPlugin));
+            .add_plugins((
+                CameraControlsPlugin,
+                ModelPreviewPlugin,
+                UserCameraDisplayPlugin::default(),
```

Comment:
I'm concerned that this would cause problems if someone wants to use the interaction plugin but not the egui widgets.

I've opened a PR that allows this to be decoupled: https://github.com/open-rmf/rmf_site/pull/247